gstd/src/async_runtime/mod.rs

```diff
@@ -49,9 +52,20 @@ pub(crate) fn locks() -> &'static mut LocksMap {
     unsafe { LOCKS.get_or_insert_with(LocksMap::default) }
 }
 
+static mut REPLY_HOOKS: Option<HooksMap> = None;
+
+fn reply_hooks() -> &'static mut HooksMap {
+    unsafe { REPLY_HOOKS.get_or_insert_with(HashMap::new) }
+}
+
 /// Default reply handler.
 pub fn record_reply() {
```

Comment:
I'd rename this now since it's not only record reply, but also calls dedicated hook for it

Reply:
Wouldn't it break backwards compatibility?
We can't guarantee someone hasn't already written

```
#[no_mangle]
extern "C" fn handle_reply() {
    gstd::record_reply();
}
```

in their program

Reply:
renamed to `handle_reply_with_hook`